integration/appaccess/appaccess_test.go

```diff
@@ -548,14 +548,64 @@ func TestTCP(t *testing.T) {
 
 			buf := make([]byte, 1024)
 			n, err := conn.Read(buf)
-			require.NoError(t, err)
+			require.NoError(t, err, buf)
```

Comment:
Why are you including `buf` here? Does that make for a more readable error message?

Reply:
Ahhh, yes, but I actually meant to remove that... will do.